apps/site/src/app/admin/participants/components/ParticipantsTable.tsx

```diff
@@ -101,10 +198,73 @@ function ParticipantsTable({
 			trackBy="_id"
 			empty={emptyMessage}
 			filter={
-				<TextFilter filteringPlaceholder="Find participants" filteringText="" />
+				<>
+					<TextFilter
+						{...filterProps}
+						countText={filteredItemsCount === 1 ? '1 participant' : `${filteredItemsCount} participants`}
+						filteringAriaLabel="Filter participants"
+					/>
+					<FormField label="Role">
+						<Select
+							data-testid="role-filter"
+							options={[ALL_ROLES].concat(roleOptions)}
+							selectedAriaLabel="Selected"
+							selectedOption={filterRole}
+							onChange={event => setFilterRole(event.detail.selectedOption as { value: string, label: string })}
+							expandToViewport={true}
+						/>
+					</FormField>
+					<FormField label="Status">
+						<Select
+							data-testid="status-filter"
+							options={[ALL_STATUSES].concat(statusOptions)}
```

Comment:
Suggestion [non-blocking]: would it be possible to show associated status icons? See the [`ApplicantFilters`](https://github.com/HackAtUCI/irvinehacks-site-2024/blob/main/apps/site/src/app/admin/applicants/components/ApplicantFilters.tsx) for reference.